.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "node"
+  - "10.11.0"
```

Comment:
This one specifies on which version of node travis should run project and test.

Set up this cause jest was failing with latest stable node version.